blogs/artificial-intelligence/mamba2-flash-attention-kernels/README.md

```diff
@@ -0,0 +1,112 @@
+---
+blogpost: true
+date: 07 December 2024
+author: Quentin Anthony, Beren Millidge
+tags: LLM, AI/ML, GenAI, PyTorch
+category: Applications & models
+language: English
+myst:
+  html_meta:
+    "description lang=en": "Mamba2 and Flash Attention Backward Kernels on AMD MI300x with ROCm"
+    "keywords": "Mamba, PyTorch, S4, S6, Mamba2, Transformer, Flash Attention, Optimization, Hardware-aware, Transformer, Attention, ROCm, Mi210, MI250, MI300, AI/ML, Generative AI"
+    "property=og:locale": "en_US"
+---
+
+
+# Training Transformers and Hybrid models on AMD Instinct MI300X Accelerators
+
```

Comment:
Added a line -- let me know how I should format this officially

Reply:
Hey @BerenMillidge , there is a new process of generating these author lines, it will automatically take whatever is in the metadata for authors and generate a line. If you want, you can create author pages for yourself and the other blog author, and the author line will link to those pages. You can take a look at the other authors and their bios to see what you would want to add.